Removed glitch media files

gideoncodenext opened the pull request:
I added a favicon image file in order to be able to remove the glitch media files. I then changed the pathways of the old glitch hrefs to reference the new favicon file. 

Fixes #63

j0n3lson commented:
(pr-nagger-bot): Friendly ping. If you are a reviewer please take a look. If you are an author, please respond to comments.

j0n3lson commented:
Hey @Steve-Leke do you want to review this one since you filled the issue?

j0n3lson commented on public/templates/demo.html:
```diff
@@ -18,7 +18,7 @@
       rel="shortcut icon"
       type="image/x-icon"
-      href="https://cdn.glitch.com/aa3eaa29-7183-4f33-b4e2-c3a30c833a25%2F31540120.jpg?v=1598133005255"
+      href="/public/assets/images/favicon.jpg"
```
This should be /relative/ to public:

`/assets/images/favicon.jpg`

Fix this every other place as well. The website root is `/public` and the `assets` folder is under that.

j0n3lson commented on public/templates/demo.html:
```diff
@@ -18,7 +18,7 @@
       rel="shortcut icon"
       type="image/x-icon"
-      href="https://cdn.glitch.com/aa3eaa29-7183-4f33-b4e2-c3a30c833a25%2F31540120.jpg?v=1598133005255"
+      href="/public/assets/images/favicon.jpg"
```
Could you also migrate the existing `png` images to the `jpg` format? Seems like `jpg` would result in faster page loads according to this article:

https://www.bluearcher.com/blog-item-jpg-vs-png-for-web#:~:text=The%20PNG%20file%20type%20holds,a%20difference%20for%20your%20audience.

Search the web for how to convert a png to a jpg.

j0n3lson replied:
Okay so from the issue you filed (#92) you mention they have to be PNGs. So should we make everything PNG instead of these new ones being jpg?

gideoncodenext replied:
@j0n3lson No. The png files that I changed to jpg's work well and the images that I imported as jpg's (the files I added in my first commit) display correctly. It is only the specific service-icon files (1-3) that need to stay as pngs. I will delete the unused png image files and the temp/patientSignUp.html in a new commit.

j0n3lson commented on temp/patientSignUp.html:
```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
```
Please delete this file. Not sure how it crept back in the source.

j0n3lson reviewed (changes requested):
The images aren't loading because the `src` attribute is incorrectly specified.

lillianchou94 commented on public/templates/patientSignUp.html:
```diff
@@ -15,7 +15,7 @@
       rel="shortcut icon"
       type="image/x-icon"
-      href="https://cdn.glitch.com/aa3eaa29-7183-4f33-b4e2-c3a30c833a25%2F31540120.jpg?v=1598133005255"
+      href="public/assets/images/favicon.jpg"
```
"/assets/images/favicon.jpg" and same for the rest of the file

gideoncodenext commented:
@j0n3lson @lillianchou94 I just made the changes y'all suggested. I changed all of the src attributes to correctly display the images and converted some of the png files to jpg's. I did not convert all of the png files -- I explain why in issue #92

j0n3lson commented:
> @j0n3lson @lillianchou94 I just made the changes y'all suggested. I changed all of the src attributes to correctly display the images and converted some of the png files to jpg's. I did not convert all of the png files -- I explain why in issue #92 

Please delete the unused images you found in this PR. You can close that issue #92 when we merge this PR.

gideoncodenext commented:
In my commit message I incorrectly specified which file I deleted. I actually deleted the old temp/patientSignUp.html file. There was no temp/patWaitingRoom.html.

j0n3lson commented:
(pr-nagger-bot): Friendly ping. If you are a reviewer please take a look. If you are an author, please respond to comments.

j0n3lson commented:
I think this is ready to go @gideoncodenext you probably have to rebase locally before it can be merged.

j0n3lson commented:
(pr-nagger-bot): Friendly ping. If you are a reviewer please take a look. If you are an author, please respond to comments.

gideoncodenext commented:
@j0n3lson @lillianchou94 Mannendri and I are a little bit confused sorting out some of these merge conflicts. Could one of y'all come to our breakout room and help us out a little bit?

Commit:
Added image files and changed PNGs to JPGs
